src/sdk/pynni/nni/gridsearch_tuner/gridsearch_tuner.py

```diff
@@ -37,37 +37,52 @@ class GridSearchTuner
 logger = logging.getLogger('grid_search_AutoML')
 
 class GridSearchTuner(Tuner):
-    '''
+    """
     GridSearchTuner will search all the possible configures that the user define in the searchSpace.
-    The only acceptable types of search space are 'choice', 'quniform', 'randint'
+    The only acceptable types of search space are ``choice``, ``quniform``, ``randint``
 
-    Type 'choice' will select one of the options. Note that it can also be nested.
+    Type ``choice`` will select one of the options. Note that it can also be nested.
 
-    Type 'quniform' will receive three values [low, high, q], where [low, high] specifies a range and 'q' specifies the interval
-    It will be sampled in a way that the first sampled value is 'low',
+    Type ``quniform`` will receive three values [``low``, ``high``, ``q``], where [``low``, ``high``] specifies a range and ``q`` specifies the interval.
+    It will be sampled in a way that the first sampled value is ``low``,
     and each of the following values is 'interval' larger than the value in front of it.
 
-    Type 'randint' gives all possible intergers in range[low, high). Note that 'high' is not included.
-    '''
+    Type ``randint`` gives all possible intergers in range[``low``, ``high``). Note that ``high`` is not included.
+
+    See Also
```

Comment:
I think this see also is not necessary. Grid search tuner users normally don't care about tuner SDK API.
I added see also to base tuner because it's name may attract users who actually want an implemented tuner to use.

Reply:
removed, thanks